lib/binaries/chrome_driver.ts

```diff
@@ -30,7 +30,13 @@ export class ChromeDriver extends Binary {
 
   suffix(ostype: string, arch: string): string {
     if (ostype === 'Darwin') {
-      return 'mac32' + this.suffixDefault;
+      // after chromedriver version 2.23, the name of the binary changed
+      // They no longer provide a 32 bit binary
+      if (parseFloat(this.version()) >= 2.23) {
```

Comment:
One suggestion I got from a colleague (@tomneyland) where we split on the `.` and check the major and minor version as ints. Maybe not the most elegant thing, but it seems pragmatic. Do you agree?